lib/pf/config.pm

```diff
@@ -687,10 +688,16 @@ sub readProfileConfigFile {
                     foreach my $field (qw(locale mandatory_fields sources filter provisioners) ) {
                         $profile->{$field} = [split(/\s*,\s*/, $profile->{$field} || '')];
                     }
-                    if ($profile_id ne 'default') {
-                        #Adding filters in profile order
+                    my $filters = $profile->{'filter'};
+                    if($profile_id ne 'default' && @$filters) {
+                        my @filterObjects;
                         foreach my $filter (@{$profile->{'filter'}}) {
-                            push @Profile_Filters, pf::factory::profile::filter->instantiate($profile_id,$filter);
+                            push @filterObjects, pf::factory::profile::filter->instantiate($profile_id,$filter);
+                        }
+                        if(defined ($profile->{filter_match_style}) && $profile->{filter_match_style} eq 'all') {
+                            push @Profile_Filters,  pf::profile::filter::all->new(profile => $profile_id, value => \@filterObjects);
```

Comment:
Just ot of curiosity... why can't we stick with some code standard... I know that may be very picky but I'm having difficulty with the fact that on two different lines, that follows each others, there's spaces after the comma, the other one nothing...
I'm doing the comment here because it is quite obvious... but that applies to a bunch of code...